Yoast/wordpress-seo
# Remove functionality saving vector length on indexable

## Summary


This PR can be summarized in the following changelog entry:

* Removes functionality that added vector length of the prominent words set on indexable.

## Relevant technical choices:

* For more context see https://github.com/Yoast/wordpress-seo-premium/pull/2611.

## Test instructions

This PR can be tested by following these steps:

* For testing instructions see https://github.com/Yoast/wordpress-seo-premium/pull/2611.

## UI changes

## Documentation

## Quality assurance


Fixes #https://github.com/Yoast/wordpress-seo-premium/issues/2593
